app/src/main/java/com/group80/uoftinder/entities/Constants.java

```diff
@@ -1,5 +1,8 @@
 package com.group80.uoftinder.entities;
```

Comment:
I suggest that this be in a separate package or not be in a package at all because this doesn't seem like an entity